### DISCUSSION TITLE
Incorrect types in docs: Atom Creators

### DISCUSSION THREAD

**Discussion by estubmo** [pmndrs/jotai#2071] [Bug report]
## Summary
The first two examples here https://jotai.org/docs/recipes/atom-creators give type errors with the latest version of Jotai and TypeScript.

## Link to reproduction

https://codesandbox.io/p/github/estubmo/jotai-docs-errors/main?file=/src/index.ts

**Comment by dai-shi** [comment]
Thanks for reporting!
Would you or anyone like to work on a fix?
